integrations/google_vertex/src/haystack_integrations/components/generators/google_vertex/captioner.py

```diff
@@ -12,16 +12,40 @@
 
 @component
 class VertexAIImageCaptioner:
+    """
+    `VertexAIImageCaptioner` enables text generation using Google Vertex AI imagetext generative model.
+
+    Authenticates using Google Cloud Application Default Credentials (ADCs).
+    For more information see the official [Google documentation](https://cloud.google.com/docs/authentication/provide-credentials-adc).
+
+    Usage example:
+    ```python
+    import requests
+
+    from haystack.dataclasses.byte_stream import ByteStream
+    from haystack_integrations.components.generators.google_vertex import VertexAIImageCaptioner
+
+    captioner = VertexAIImageCaptioner(project_id=project_id)
+
+    image = ByteStream(data=requests.get("https://raw.githubusercontent.com/silvanocerza/robots/main/robot1.jpg").content)
```

Comment:
Let's store the assets we're using in the integration folder like we do for Google AI integration and change this URL maybe.

https://github.com/deepset-ai/haystack-core-integrations/tree/main/integrations/google_ai/example_assets

Reply:
done